internal/attestation/measurements/measurements.go

```diff
@@ -18,45 +18,76 @@ import (
 	"net/url"
 
 	"github.com/edgelesssys/constellation/v2/internal/attestation/vtpm"
+	"github.com/edgelesssys/constellation/v2/internal/cloud/cloudprovider"
 	"github.com/edgelesssys/constellation/v2/internal/sigstore"
 	"gopkg.in/yaml.v2"
 )
 
 // Measurements are Platform Configuration Register (PCR) values.
 type Measurements map[uint32][]byte
```

Comment:
nit: A common thing in Go would be to call this just `M`, like in `testing.T`, as it its the only exported type in the package and the package name is already descriptive enough...

Reply:
Thanks for the encouragement! I have thought about it but didn't find any instances of it in our code base, so I decided against it. I too like the short form better, so that clients do not stutter.